internal/server/data/migrations.go

```diff
@@ -895,3 +896,16 @@ func addAllowedDomainsToProvidersTable() *migrator.Migration {
 		},
 	}
 }
+
+func fixDefaultOrgCreatedByMigration() *migrator.Migration {
+	return &migrator.Migration{
+		ID: "2022-11-10T17:30",
+		Migrate: func(tx migrator.DB) error {
+			stmt := `
+				UPDATE organizations SET created_by=1 WHERE created_by is null;
+				UPDATE organizations SET domain='' WHERE domain is null`
```

Comment:
This `SET domain=''` will fail if there are multiple organizations with a null domain. That shouldn't ever be the case because the only way to get a null in that field was from the migration that created a single org.